reggie.py

```diff
@@ -3238,6 +3252,18 @@ def LoadLevel_NSMBW(self, levelData, areaNum):
             self.scene.addItem(com)
             com.UpdateListItem()
 
+    def OpenPuzzle(self, tilesetIndex = None):
+        if tilesetIndex is not None:
+            if self.objAllTab.isTabEnabled(tilesetIndex):
+                tilesets = [globals_.Area.tileset0, globals_.Area.tileset1, globals_.Area.tileset2, globals_.Area.tileset3]
+                tilesetDir = " " + globals_.gamedef.GetGamePath() + "/Texture/" + tilesets[tilesetIndex] + ".arc"
```

Comment:
You probably want to use `os.path.join` to make sure the path separators are correct.